src/status_im/setup/oops.cljs

```diff
@@ -0,0 +1,20 @@
+(ns status-im.setup.oops
+  (:require [oops.config]))
+
+(defn setup!
+  "Change oops defaults to warn and print instead of throwing exceptions during
+  development."
+  []
+  (oops.config/update-current-runtime-config!
+   merge
+   {:error-reporting              :console
+    :expected-function-value      :warn
+    :invalid-selector             :warn
+    :missing-object-key           :warn
+    :object-is-frozen             :warn
+    :object-is-sealed             :warn
+    :object-key-not-writable      :warn
+    :unexpected-empty-selector    :warn
+    :unexpected-object-value      :warn
+    :unexpected-punching-selector :warn
+    :unexpected-soft-selector     :warn}))
```

Comment:
I think this should also affect production. The app has many ignored warnings, so this setting may create unexpected crashes on prod builds

Reply:
Good point @ulisesmac.

From the docs:

> By default, all diagnostics code is elided in :advanced builds and oops produces code similar to hand-written aget usage (without any safety-checks).

So that's why I didn't bother checking the environment, because oops already disables everything on advanced compilation.

But since you asked, I can change the code to only call setup in non-release builds.

Reply:
> So that's why I didn't bother checking the environment, because oops already disables everything on advanced compilation.

I didn't know it.

It can be kept as is then :+1: